R/RLearner_regr_randomForest.R

```diff
@@ -1,19 +1,44 @@
-#' regression using randomForest.
+#' @title RandomForest regression learner.
 #'
-#' a mlr learner for regrssion tasks using \code{\link[randomForest]{randomForest}}.
+#' @description
+#' mlr learner for regression tasks using \code{\link[randomForest]{randomForest}}.
 #'
-#' if \code{predict.type = "se"} the \code{se.method} (by default \dQuote{jackknife})
-#' is estimated, using the methods described in Sexton and Laake (2009).
+#' This doc page exists, as we added additional uncertainty estimation functionality
+#' (\code{predict.type = "se"}) for the randomForest, which is not provided by the underlying package.
 #'
-#' If \code{se.method = "bootstrap"} the standard error of a prediction is estimated by bootstrapping the random forest, where the number of bootstrap replicates and the number of trees in the ensemble are controlled by \code{se.boot} and \code{ntree.for.se} respectively, and then taking the standard deviation of the predictions.
+#' Currently implemented methods are:
 #'
-#' If \code{se.method = "jackknife"}, the default, the standard error of a prediction is estimated by computing the jackknife-after-bootstrap, the mean-squared difference between the prediction made by only using trees which did not contain said observation and the ensemble prediction.
+#' \itemize{
+#' \item If \code{se.method = "jackknife"}, the default, the standard error of a prediction
+#'   is estimated by computing the jackknife-after-bootstrap, the mean-squared difference between
+#'   the prediction made by only using trees which did not contain said observation and
+#'   the ensemble prediction.
+#' \item If \code{se.method = "bootstrap"} the standard error of a prediction is estimated by
+#'   bootstrapping the random forest, where the number of bootstrap replicates and the number of
+#'   trees in the ensemble are controlled by \code{se.boot} and \code{se.ntree} respectively,
+#'   and then taking the standard deviation of the bootstrap predictions. The "brute force" bootstrap
+#'   is executed when \code{ntree = se.ntree}, the latter of which controls the number of trees in the
+#'   individual random forests which are bootstrapped. The "noisy bootstrap" is executed when \code{se.ntree < ntree}
+#'   which is less computationally expensive. A Monte-Carlo bias correction may make the latter option
+#'   prefarable in many cases. Defaults are \code{se.boot = 50} and \code{se.ntree = 100}.
```

Comment:
Maybe emphasize that only the `sd` is calculated by the bootstrap and the `mean` is the prediction of the original forest